prisma/migrations/20240326181608_api_key_sha256/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `apiKey` on the `users` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[apiKeyHash]` on the table `users` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "users.apikey_unique";
+
+-- AlterTable
+ALTER TABLE "users" DROP COLUMN "apiKey",
```

Comment:
Yea I guess my point was, if users had generated keys, this would break their existing keys , which isn't ideal. Yes they can make a new one but their existing one would suddenly stop working.

It's probably more of an academic exercise to migrate the existing keys instead of just wiping them out, since the feature isn't widely adopted.

Reply:
Yes, I definitely would have reconsidered this if at least one stacker which wasn't me generated new keys